data/maps/AncientTomb/scripts.inc

```diff
@@ -25,7 +25,7 @@ AncientTomb_EventScript_238FEE:: @ 8238FEE
 	return
 
 AncientTomb_MapScript1_238FF2: @ 8238FF2
-	call_if_unset FLAG_SYS_BRAILLE_FLY, AncientTomb_EventScript_238FFC
+	call_if_unset FLAG_SYS_BRAILLE_REGISTEEL_HM, AncientTomb_EventScript_238FFC
```

Comment:
Not sure I like this name, maybe make it `REGISTEEL_PUZZLE`?

Reply:
I wanted to keep the reference to HM because the only thing that changed across versions for these two puzzles was the specific HM used. If you still want to change them, I'll do so.

Reply:
I'll just call it `REGISTEEL_PUZZLE_COMPLETED`